source/neuropod/BUILD

```diff
@@ -40,21 +40,32 @@ cc_library(
 )
 
 cc_binary(
-    name = "libneuropod.so",
+    name = "libneuropod_orig.so",
     linkopts = select({
         "@bazel_tools//src/conditions:darwin": ["-Wl,-rpath,@loader_path"],
         "//conditions:default": ["-Wl,-rpath,$$ORIGIN"],
     }),
     linkshared = True,
     linkstatic = True,
-    visibility = [
-        "//visibility:public",
-    ],
     deps = [
         ":neuropod_impl",
     ],
 )
 
+# This genrule changes the dylib id for libneuropod.so on mac
+genrule(
+    name = "change_dylib_id",
```

Comment:
Please confirm if I understand it right:

this rule is called when any lib/bin depends on libneuropod.so and as result command install_name_tool is called on this lib/bin during build?

Reply:
Correct!